docs/cloud-provider-specifics.md

```diff
@@ -155,6 +155,8 @@ To integrate Heptio Ark with GCP, you should follow the instructions below to cr
 
 In the Ark root directory, run the following to first set up namespaces, RBAC, and other scaffolding:
 
+When using Google Kubernetes Engine (GKE), be sure to make your current IAM user a cluster-admin. See [their docs][22] for more information.
```

Comment:
Personally I would probably move this above the prior line, and add a brief explanation of why this is necessary (i.e. on GKE you are not cluster-admin by default and this prevents you from creating the RBAC resources in 00-prereqs.yaml

Reply:
Thanks for that feedback - does the new commit help?

Reply:
WFM. Would appreciate a last glance from @Bradamant3